packages/ringcentral-widgets-test/test/integration-test/AllCallsPage/AllCalls.spec.js

```diff
@@ -0,0 +1,145 @@
+import * as mock from 'ringcentral-integration/integration-test/mock';
```

Comment:
as for integration tests for All Calls Page, you can put it in to catelog calls/